EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -7,22 +7,36 @@
  *******************************************************************************/
 package de.hpi.unicorn.application.pages.input.generator;
 
+import com.google.common.io.Files;
+import de.hpi.unicorn.application.pages.export.AJAXDownload;
 import de.hpi.unicorn.application.pages.input.generator.attributeInput.AttributeInput;
 import de.hpi.unicorn.application.pages.input.generator.validation.DateRangeValidator;
 import de.hpi.unicorn.attributeDependency.AttributeDependencyManager;
 import de.hpi.unicorn.event.EapEventType;
 import de.hpi.unicorn.event.attribute.TypeTreeNode;
+import de.hpi.unicorn.importer.json.JsonExporter;
+import de.hpi.unicorn.importer.json.JsonImporter;
+import org.apache.log4j.Logger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.log4j.Logger'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)